### ISSUE TITLE
`respository` typo?

### ISSUE THREAD

**Issue by mishak87** [mayflower/docker-ls#4]
Is `respository` instead of repository [in code](https://github.com/mayflower/docker-ls/search?utf8=%E2%9C%93&q=respository) intentional or just typo?

``` sh
> docker-ls tags # show me help please 
usage: docker-ls tags [options] <respository>
...
```

**Comment by DirtyHairy** [comment]
Hi @mishak87!

Nope, just a combination of typo + autocomplete. Thanks for the pointer.

Cheers
-Christian

**Comment by DirtyHairy** [comment]
Fixed in 49c9644 .
